app/views/assigned_applications/index.html.erb

```diff
@@ -6,44 +6,52 @@
     <p><%=t '.your_assignments', count: @applications.count %></p>
   </div>
 </div>
+
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full govuk-!-margin-bottom-5">
-    <%= button_to(
-          'Get next application',
-          get_next_assigned_applications_path,
-          method: :post,
-          class: 'govuk-button'
-        ) %>
+    <%= form_with url: get_next_assigned_applications_path, method: :post do |f| %>
```

Comment:
Forms default to post so I don't think we need to specify it.

Reply:
Just thinking as well why not just keep the button with the post? there is only one element in this form so its functially in the same. I think we could have the link outside the form as it is not using the form object and then just wrap them in a row.